cfn-resources/alert-configuration/README.md

```diff
@@ -0,0 +1,64 @@
+# MongoDB::Atlas::AlertConfiguration
+
+## Description
+Returns and edits the conditions that trigger alerts.
+
+## Attributes & Parameters
+
+Please consult the [Resource Docs](docs/README.md)
+
+## Unit Testing Locally
+
+The local tests are integrated with the AWS `sam local` and `cfn invoke` tooling features:
+
+```
+sam local start-lambda --skip-pull-image
+```
+then in another shell:
+```bash
+repo_root=$(git rev-parse --show-toplevel)
+source <(${repo_root}/quickstart-mongodb-atlas/scripts/export-mongocli-config.py)
+cd ${repo_root}/cfn-resources/alert-configuration
+./test/alert-configuration.create-sample-cfn-request.sh YourProjectID  > test.request.json 
+echo "Sample request:"
+cat test.request.json
+cfn invoke CREATE test.request.json 
+cfn invoke DELETE test.request.json 
+```
+
+Both CREATE & DELETE tests must pass.
+
+## Installation
+TAGS=logging make
+cfn submit --verbose --set-default
+
+## Cloudformation Examples
+
+Please see the [CFN Template](test/alert-configuration.sample-template.yaml) for example resource
```

Comment:
Can you add this template to example folder and link to there instead? 
https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/examples

Reply:
@Zuhairahmed - We have fixed the review comments.Please review it